lib/logasm/preprocessors/strategies/mask_strategy.js

```diff
@@ -0,0 +1,59 @@
+const { isObject } = require('../../object_helpers');
+
+const WILDCARD_SYMBOL = '~';
+const MASKED_VALUE = '*****';
+
+class MaskStrategy {
```

Comment:
Btw, you can also encapsulate objects like
```js
function createMaskStrategy(whitelistedFields) {
  const somePrivateFunc = ...;
  const process = function(data, path) {...}

  return {process: process};
}
module.exports = createMaskStrategy;
```
We used this style in Medra a lot. This makes `this.` redundant so you don't have to worry about scoping at all. We found it quite useful.

Not forcing this change though.